src/Chainweb/Utils.hs

```diff
@@ -651,6 +652,10 @@ encodeB64UrlNoPaddingText :: B.ByteString -> T.Text
 encodeB64UrlNoPaddingText = T.dropWhileEnd (== '=') . T.decodeUtf8 . B64U.encode
 {-# INLINE encodeB64UrlNoPaddingText #-}
 
+encodeB64UrlNoPadding :: B.ByteString -> B.ByteString
```

Comment:
Do we have a roundtrip for this?

Reply:
I ended up not using it in this PR. Going to remove it.